ofl/playwritecu/article/ARTICLE.en_us.html

```diff
@@ -0,0 +1,134 @@
+<p>
+ Literacy education in Cuba is notable for its distinctive approach: students are introduced to two different styles of letters from the outset, one for reading and the other for writing.
+ Textbooks feature print-style, typographic letters represented by either geometric or grotesque sans serifs for reading.
+ These styles are consistent across all workbooks and textbooks, with the exception of those focused on writing and calligraphy.
+ For these, a fully-joined, cursive style is used, drawing inspiration from calligraphic models from the
+ <a href="https://fonts.google.com/specimen/Playwrite+US+Trad">
+  United States
+ </a>
+ , notably the Palmer Method, created by Austin Palmer , and the Zaner-Bloser method, developed by Charles Paxton Zaner and his partner Elmer Ward Bloser.
+</p>
+<p>
+ Playwrite Cuba is a variable font with a weight range from Thin (100) to Regular (400), and supports over 150 Latin-based languages.
+</p>
+<p>
+ To contribute, see
+ <a href="https://github.com/TypeTogether/Playwrite/">
+  github.com/TypeTogether/Playwrite
+ </a>
+ .
+</p>
+<hr/>
+<img src="Playwrite-Cuba-1.png"/>
+<h4>
+ Playwrite Cuba characteristics
+</h4>
+<p>
+ This style is a very slanted continuous cursive that draws inspiration from the Zaner-Bloser and D'Nealian models.
+ It features cursive uppercase letters, some of which are quite complex, such as 'I', 'J', and 'G'.
+ The lowercase letters have medium-length extenders with loops, present even in letters like 'p' and 'q', and are formed using slow strokes.
+ Notably, this style includes an italic-style 'r' and a 'z' with a flat top and a loop.
+</p>
+<img src="Playwrite-Cuba-3.png"/>
+<p>
+ Playwrite Cuba appears in font menus with a two-letter country code ‘CU’,
+ <i>
+  Playwrite CU
+ </i>
+ , and features four styles: Thin, ExtraLight, Light, and Regular.
```

Comment:
There is a space just before the comma (between CU and comma)

Reply:
We've decided to merge them as they are now and update them later after Marc changes the HTML auto-formatter